packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -127,6 +136,16 @@ const TradersHub = observer(() => {
         return <OrderedPlatformSections is_cfd_visible={false} is_options_and_multipliers_visible={true} />;
     };
 
+    const ShowRealAccountCreationBanner = () => {
```

Comment:
@mitra-deriv please, move components creation outside of the parent component or just use ternary operator to show/hide `<RealAccountCreationBanner />` below:

`{((no_CR_account && !is_eu_user) || (no_MF_account && is_eu_user)) ? <RealAccountCreationBanner /> : null}`

Also, I'm not sure you need to wrap the component with `div`, because `RealAccountCreationBanner` component is already wrapped by `div`

Reply:
@mitra-deriv better to lazyload this banner